mitodl/open-discussions
# Added email and email_optin fields to user API (#447)

#### What are the relevant tickets?
Resolves #447

#### What's this PR do?
Adds `email` and `email_optin` as optional fields to preserve backwards compatibility

#### How should this be manually tested?
- Verify you don't see any errors in MM profile sync after updating your profile
- Verify you can create a new user account via the API with and without the above values
- Verify you can update an existing user with/without the above values passed
